config/initializers/assets.rb

```diff
@@ -12,3 +12,4 @@
 # application.js, application.css, and all non-JS/CSS in the app/assets
 # folder are already added.
 # Rails.application.config.assets.precompile += %w( admin.js admin.css )
+Rails.application.config.assets.precompile += ["budget.css"]
```

Comment:
This is not needed if you import the budgets stylesheets in the application stylesheet.
Adding two stylesheets, means the browser will need to download another file, which will affect performance.
It is probably not significant in this app, but is a good practice for when you will be working on bigger apps, with many more assets.

Reply:
Done in sha: a2fa585